.buildkite/scripts/health-report-tests/main.sh

```diff
@@ -1,9 +1,5 @@
 #!/usr/bin/env bash
 set -eo pipefail
 
-# TODO:
-#  if branch is specified with X.Y, pull branches from ACTIVE_BRANCHES_URL="https://raw.githubusercontent.com/elastic/logstash/main/ci/branches.json", parse and use
-#  build Logstash from specificed (ex: 8.x -> translates to 8.latest, 8.16) branch, defaults to main
-#  install requirements of the python package and run main.py
-
-
+python3 -mpip install -r .buildkite/scripts/health-report-tests/requirements.txt
```

Comment:
FYI: python modules such as `pip` missing in [Logstash CI BK agent](https://github.com/elastic/logstash/blob/main/.buildkite/health_report_tests_pipeline.yml#L4), so I will have a follow up for aligning on default agent (tested with [PR](https://github.com/elastic/logstash/pull/16524), [result CI](https://buildkite.com/elastic/logstash-health-report-tests-pipeline/builds/39))